src/main/java/com/mg105/entities/Inventory.java

```diff
@@ -0,0 +1,61 @@
+package com.mg105.entities;
+import java.util.ArrayList;
+
+public class Inventory {
+    final static int inventoryLimit = 10;
+    private static ArrayList<Item> items = new ArrayList<>();
+
+    public static int getInventoryLimit(){
+        return inventoryLimit;
+    }
```

Comment:
What is the point of this? By the `final` in its type declaration `inventoryLimit` can never changed.